.standard.yml

```diff
@@ -15,3 +15,5 @@ ignore:
     - Layout/AlignHash
   - 'app/**/*':
     - Naming/AccessorMethodName
+  - 'app/controllers/advertisements_controller.rb':
+    - Performance/InefficientHashSearch
```

Comment:
Had to add this bc Standard was trying to correct something and if the correction was used, an error was thrown.